docs/kubernetes/README.md

```diff
@@ -6,6 +6,16 @@
 - Dedicated Namespace to run the jobs.
 - RWX volume to store the evaluation results (check `volume.yml` for inspiration).
 
+### Defining secrets
+
+The Job automatically references to a secret called `evaluation-secret` which is configured to pass the values on as environment Variables. The following key is required to be created.
+
+- `PROVIDER_TOKEN` - contains the API tokens for different providers. E.g.: `openrouter:abcdefgh1234,custom-provider:abcdefgh1234`
+
+```bash
+kubectl --namespace eval-dev-quality create secret generic evaluation-secret --from-literal='PROVIDER_TOKEN='
```

Comment:
Should we run this command within the `kubernetes` runtime automatically to carry over the tokens from the ENV? I guess passing the arguments from the CLI should work automatically, right? I kind of wonder why it does not work out of the box. If you have them in the environment, then they should be read into the CLI arguments... and then be passed to the containers running on kubernetes, no?